### PR TITLE
Guide users to the WinUI repo

### PR DESCRIPTION
**Pull request by btueffers**

### CODE REVIEW

File: docs/contributor-guide.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ points if there's a gist or existing repo we can look at with you.
 3. **[Make a feature proposal](https://github.com/microsoft/WindowsAppSDK/issues/new?assignees=&labels=feature+proposal&template=feature-proposal.md&title=)**<br>
    What components do you want to see in the Windows App SDK? For example you can propose a new API on an existing type, or an idea for a new API.
    It's fine if you don't have all the details: you can start with a summary and rationale.
+   
+4. **[WinUI topics](https://github.com/microsoft/microsoft-ui-xaml)**<br>
+   Check out the WinUI repos amazing guide to [general contributing](https://github.com/microsoft/microsoft-ui-xaml/blob/main/CONTRIBUTING.md) as well as [contributing feedback and requests](https://github.com/microsoft/microsoft-ui-xaml/blob/main/CONTRIBUTING_feedback_and_requests.md)!
```
Comment (andrewleader) [inline docs/contributor-guide.md:23]: Just missing an apostrophe :) 

```suggestion
   Check out the WinUI repo's amazing guide to [general contributing](https://github.com/microsoft/microsoft-ui-xaml/blob/main/CONTRIBUTING.md) as well as [contributing feedback and requests](https://github.com/microsoft/microsoft-ui-xaml/blob/main/CONTRIBUTING_feedback_and_requests.md)!
```

**Review by andrewleader — APPROVED** [review]
Looks great 👍 just one tiny grammar change!

**Comment by riverar** [comment]
Why would you refer to a separate repo's contributor's guide? That's just confusing given it contains all their terminology and ISMs that don't really apply here.

**Comment by andrewleader** [comment]
@riverar we're suggesting people file WinUI issues in the WinUI repo, maybe we need to make that even clearer by simply saying


**WinUI topics**
Please file WinUI issues/questions/etc in the WinUI repository: microsoft/microsoft-ui-xaml

**Comment by riverar** [comment]
@andrewleader Oh! I completely missed the linked title. Agreed, anything you can add to clarify that would be fantastic. Maybe just one more sentence like you suggested.

**Comment by btueffers** [comment]
@andrewleader and @riverar I made a slight change to incorporate your feedback!

File: docs/contributor-guide.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ points if there's a gist or existing repo we can look at with you.
    It's fine if you don't have all the details: you can start with a summary and rationale.
    
 4. **[WinUI topics](https://github.com/microsoft/microsoft-ui-xaml)**<br>
-   Check out the WinUI repo's amazing guide to [general contributing](https://github.com/microsoft/microsoft-ui-xaml/blob/main/CONTRIBUTING.md) as well as [contributing feedback and requests](https://github.com/microsoft/microsoft-ui-xaml/blob/main/CONTRIBUTING_feedback_and_requests.md)!
+   Please file WinUI issues and feedback in the [WinUI repository](https://github.com/microsoft/microsoft-ui-xaml). Check out the their amazing guide to [general contributing](https://github.com/microsoft/microsoft-ui-xaml/blob/main/CONTRIBUTING.md) as well as [contributing feedback and requests](https://github.com/microsoft/microsoft-ui-xaml/blob/main/CONTRIBUTING_feedback_and_requests.md)!
```
Comment (andrewleader) [inline docs/contributor-guide.md:23]: Looks good! Ended up with an extra "the" though :P

```suggestion
   Please file WinUI issues and feedback in the [WinUI repository](https://github.com/microsoft/microsoft-ui-xaml). Check out their amazing guide to [general contributing](https://github.com/microsoft/microsoft-ui-xaml/blob/main/CONTRIBUTING.md) as well as [contributing feedback and requests](https://github.com/microsoft/microsoft-ui-xaml/blob/main/CONTRIBUTING_feedback_and_requests.md)!
```

**Review by riverar — APPROVED** [review]

**Pull request merged**